pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     # "rdkit-pypi>=2022.3.5",
     "jarvis-tools>=2023.12.12",
     # To avoid module 'numpy.linalg._umath_linalg' has no attribute '_ilp64' in Colab
```

Comment:
@timurbazhirov what is the numpy version requirement for Colab? There are some breaking (minor) changes on numpy v2 (recently released). For now, we may pin to numpy v1. Also, numpy now requires Python v3.9 or above. Tests are failing on v3.8, because it cannot install numpy v1.26.0. If Colab compatibility allows, we may pin to v1.24.4.

Reply:
Maybe let's just use "numpy<2" and let pip figure it out?

Reply:
Including @knc6 - any insight regarding the numpy version for Colab?

Reply:
The root of the problem appears to be presence of multiple numpy versions in the system https://github.com/numpy/numpy/issues/25150 Colab comes with preinstalled set of python packages, including numpy. As of now, Colab uses the latest release of numpy v1. So, with current range specification, we are good for now. But the problem might come back when Colab upgrades to numpy v2 and we are not. We can fix our code in a separate PR, so that we can use numpy v2 as well.